.github/pages/hello-world.md

```diff
@@ -35,6 +35,39 @@ This downloads the Fashion-MNIST training and test dataset and tells Jina to ind
 Then it randomly samples images from the test set as queries and asks Jina to retrieve relevant results.
 The whole process takes about 1 minute.
 
+<br><br>
+
+#### Use jina hub Executors
+
+
+ You can run the `jina hello fashion` demo using a different embedding method.
+ As an example, you can use  [ImageTorchEncoder](https://github.com/jina-ai/executor-image-torch-encoder)
+  changing in `jina/helloworld/fashion/app.py` the flow
```

Comment:
Let's explain this part in the following step.

1. create your own project by `jina hello fork fashion <your_project_folder>`
2. edit `<your_project_folder>/app.py` and change Line 74 to 79

Reply:
Plus, we need a one-sentence explanation about `jinahub+docker://`

Reply:
Done